sbin/run_e2e_tests.sh

```diff
@@ -99,6 +99,13 @@ sleep 1
 $SBIN/start_sidecar.sh &>proc3.out &
 sleep 1
 
+CONTRACTS_ENV=$CONTRACTS_DIR/.env
+until [ -f $CONTRACTS_ENV ]; do
+  echo "waiting for contracts/.env to be generated..."
+  sleep 4
+done
+cat .deployments.env >>$CONTRACTS_ENV
+
```

Comment:
i think it makes more sense to put this in `deploy_l1_contracts.sh` since we copy `.contracts..env` and generate the `.deployments.env` there